Doc/library/ssl.rst

```diff
@@ -714,7 +714,7 @@ Constants
    Selects SSL version 2 as the channel encryption protocol.
 
    This protocol is not available if OpenSSL is compiled with the
-   ``OPENSSL_NO_SSL2`` flag.
+   ``no-ssl2`` option (``OPENSSL_NO_SSL2`` flag defined).
```

Comment:
I'm not sure if we need to mention the C defines at all.

Reply:
Yup I'm with you here, I'd stick with the config options only for brevity myself. 👍

Reply:
```suggestion
   ``no-ssl2`` option.
```